### ISSUE TITLE
Wrong projector preview resolution

### ISSUE THREAD

**Issue by donovaly** [OpenSlides/OpenSlides#3053]
- change the default resolution for the standard projector from 1024 x 768 to the more common 1280 x 960 pixel.

result: the projector preview is now cutoff at the right side:
![localhost 8000 2017-03-05 01-53-37](https://cloud.githubusercontent.com/assets/1828501/23583504/cd851580-0146-11e7-9951-e28b106edc82.jpg)

(my computer resolution (also valid for the browser) is 1920 x 1080.

**Comment by donovaly** [comment]
During my first assembly with OS I noticed this as important point: I had 2 different projectors. Both have a different resolution, one was an old one (looked like solen from a museum ;-) ) with 1024 x 768 pixels resolution. This one worked fine. The newer one had a higher 16 : 9 resolution. To avoid artifacts I tried to change the projector resolution and stumbled again over this bug.

Nevertheless I still think that a higher resolution than current should be the default.
